### ISSUE TITLE
No Encoder cutouts on Rev3 Top Plates

### ISSUE THREAD

**Issue by DreamShark-Bytes** [splitkb/kyria#16]
All 4 files for Rev3's Top Plates don't have Encoder cutouts. 
1. Kyria rev3 Top Plate - 0.1mm Kerf.dxf
2. Kyria rev3 Top Plate - No Kerf
3. Kyria rev3 Top Plate - No Puck - 0.1mm Kerf.dxf
4. Kyria rev3 Top Plate - No Puck - No Kerf.dxf

Here is an example of one file:
![2023-04-14_23-35-45](https://user-images.githubusercontent.com/70111379/232183130-66361a66-eaf1-4957-b146-98f20225f6f9.png)

**Comment by thomasbaart** [comment]
That is correct. Since the rev3 has hotswap capabilities, you'll want to add the cutouts yourself. Adding cutouts to all positions by default would make switches on those locations wobbly.

It's not an ideal solution, but I think it's the best compromise.

**Comment by autoferrit** [comment]
can this be done at all with the splitkb kit version if we want to use choc switches? would like to be able to use encoders with it.
